src/components/dialog/Dialog.vue

```diff
@@ -269,6 +270,10 @@ export default {
                 this.$refs.confirmButton.$el.focus()
             }
         })
+    },
+
+    beforeDestroy() {
+        clearTimeout(this.timeOutID)
```

Comment:
@Arooba-git This should not be necessary. Because `timeOutID` is set only if `Dialog` has programmatically been opened (`this.programmatic` is `true`) and a programmatically opened `Dialog` won't be destroyed until the timeout expires. Please see the line 228.

Reply:
@Arooba-git I have reverted the changes in your fork.